source/extensions/filters/http/local_ratelimit/local_ratelimit.cc

```diff
@@ -44,6 +48,25 @@ FilterConfig::FilterConfig(
   if (per_route && !config.has_token_bucket()) {
     throw EnvoyException("local rate limit token bucket must be set for per filter configs");
   }
+
+  for (const auto& descriptor : config.descriptors()) {
+    // If there is no limit or descriptors configured, then there is no config
+    // to rate limit on.
+    if (descriptor.entries().size() == 0) {
+      ENVOY_LOG(warn, "wrong config supplied for local rate limit");
```

Comment:
i think if `per_route` is true, we should throw an error here instead of just warning.

Reply:
+1, same below

Reply:
Thanks..Fixed